File: dataset/human/c/file_2641.c
#include<stdio.h>
int main()
{
 char s[3];

 
  
 scanf("%s",&s);

 if(s[0]=='a'){
  if((s[1]=='b'&&s[2]=='c')||(s[1]=='c'&&s[2]=='b')){
    printf("Yes");
  }
   else printf("No");
 }
    if(s[0]=='b'){
  if((s[1]=='a'&&s[2]=='c')||(s[1]=='c'&&s[2]=='a')){
    printf("Yes");
  }
      else printf("No");
 } 
  if(s[0]=='c'){
  if((s[1]=='a'&&s[2]=='b')||(s[1]=='b'&&s[2]=='a')){
    printf("Yes");
  }
   else printf("No");
 }
   return 0;
}

File: dataset/human/java/file_2641.java
import java.util.*;
// warm-up
public class Main {

	static void solve() {
		Scanner sc = new Scanner(System.in);
		int[] c = new int[256];
		boolean ok = true;
		String a = sc.next(), b = sc.next();
		for (char ch : b.toCharArray()) c[ch]++;
		for (char ch : a.toCharArray()) {
			if (c[ch]>0) c[ch]--;
			else ok=false;
		}
		if (ok) ok = a.length()<b.length();
		if (!ok) {
			StringBuilder f = new StringBuilder();
			StringBuilder g = new StringBuilder();
			char[] d = a.toCharArray(), e = b.toCharArray();
			Arrays.sort(d); Arrays.sort(e);
			for (char ch : d) f.append(ch);
			for (char ch : e) g.append(ch);
			ok=f.compareTo(g.reverse())<0;
		}
		System.out.println(ok ? "Yes" : "No");
		sc.close();		
	}

	public static void main(String args[]) {
		solve();
	}

}
